service/handlers.go

```diff
@@ -28,6 +28,9 @@ func (rh *RouteHandler) ListWebhooks(w http.ResponseWriter, r *http.Request) (in
 	objs, err := apiClient.GenericObject.List(&client.ListOpts{})
 	response := []model.Webhook{}
 	for _, obj := range objs.Data {
+		if obj.Type == "register" {
```

Comment:
Let's do this a little differently:
1. When creating the generic object, set kind=webhookReceiver
2. When getting the list of genericObjects from cattle, add a filter for kind=webhookReceiver
To do this, you'll have to change the authorization of genericObject.kind to allow it to be set on create. So that field will become `'crd'`.